### DISCUSSION TITLE
How can I implement the fancy tooltips that mkdocs material uses for their own docs?

### DISCUSSION THREAD

**Discussion by pkeilbach** [squidfunk/mkdocs-material#7607] [Q&A] [answered]
I stumbled over the [tooltips feature](https://squidfunk.github.io/mkdocs-material/reference/tooltips/) to create a glossary, and this is exactly what I want.

But I was wondering how I can implement a tooltip like the following (that mkdocs material uses themselves):

![image](https://github.com/user-attachments/assets/59002cfa-b60e-4ea2-9c91-90f373cece87)

I was checking the source code of the page to see how it is done, and it is just [referencing](https://github.com/squidfunk/mkdocs-material/blob/cf2b39d1c2858c53c17e4b5ce8ee33a5c94b6d98/docs/reference/tooltips.md?plain=1#L33) the original section there.

When I implement it like this, all I get is a regular link to the corresponding markdown section (which is somehow expected because it is the regular syntax for markdown links), but no tooltip.

So I was wondering what am I missing and how mkdocs material is doing it themselves. Am I missing some configuration?

**Comment by squidfunk** [comment, marked as answer]
Those nice little fancy tooltips are called [Instant previews](DemoConfig) and are available to [our sponsors](https://github.com/sponsors/squidfunk) as part of [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) ☺️

**Reply by pkeilbach** [reply]
Cool stuff, @squidfunk 🙌🏼 

Also, I wanted to say, I use mkdocs material for a [university course](https://pkeilbach.github.io/htwg-practical-nlp/) I am teaching and it is the perfect tool for that 😍 

Thank you!
